discovery_child_development/pipeline/models/binary_classifier/04b_train_distilbert_classifier_sweeps.py

```diff
@@ -0,0 +1,139 @@
+"""
+Run the binary classifier using distilbert-base-uncased.
+
+Usage:
+
+python discovery_child_development/pipeline/models/binary_classifier/04b_train_distilbert_classifier_sweeps.py
+
+Optional arguments:
+--production : Determines whether to create the embeddings for the full dataset or a test sample (default: True)
+--count : Determines the number of runs (default: 20)
+"""
+import wandb
+import numpy as np
+import argparse
+from nesta_ds_utils.loading_saving import S3
+from transformers import TrainingArguments
+from discovery_child_development.getters.binary_classifier.embeddings_hugging_face import (
+    get_embeddings,
+)
+from discovery_child_development.utils.huggingface_pipeline import (
+    load_model,
+    load_trainer,
+)
+from discovery_child_development.utils import wandb as wb
+from discovery_child_development import (
+    logging,
+    S3_BUCKET,
+    config,
+    binary_config,
+)
+from transformers import set_seed
+
+# Set up
+S3_PATH = "models/binary_classifier/"
+VECTORS_PATH = "data/labels/binary_classifier/vectors/"
+VECTORS_FILE = "distilbert_sentence_vectors_384_labelled"
+SEED = config["seed"]
+# Set the seed
+set_seed(SEED)
+
+if __name__ == "__main__":
+    # Set up the command line arguments
+    parser = argparse.ArgumentParser(
+        description="Script to run with command line arguments."
+    )
+
+    parser.add_argument(
+        "--production",
+        type=bool,
+        default=False,
+        help="Do you want to run the code in production? (default: False)",
+    )
+
+    parser.add_argument(
+        "--count",
+        type=bool,
+        default=False,
+        help="Choose number of runs (default: 20)",
+    )
+    # Parse the arguments
+    args = parser.parse_args()
+    logging.info(args)
+
+    if not args.production:
+        VECTORS_FILE = VECTORS_FILE + "_test"
```

Comment:
Hey @j-gillam had to delete these two lines for the script to work
(otherwise it looks for a file named 
`data/labels/binary_classifier/vectors/distilbert_sentence_vectors_384_labelled_test_test_train.pkl`)

Reply:
Yeah that file currently doesn't exist in the newest iteration but can do if you run the pipeline with --production False. But happy for it to be removed as I don't think it adds anything!